shell/common/node_bindings.cc

```diff
@@ -531,7 +543,8 @@ node::Environment* NodeBindings::CreateEnvironment(
   // renderer process.
   is.allow_wasm_code_generation_callback = AllowWasmCodeGenerationCallback;
 
-  if (browser_env_ == BrowserEnvironment::kBrowser) {
+  if (browser_env_ == BrowserEnvironment::kBrowser ||
+      browser_env_ != BrowserEnvironment::kUtility) {
```

Comment:
Is this correct? It looks like a copy/paste error to have `!=` instead of `==` here.

Reply:
Oops, yeah.